app/components/search/search.component.ts

```diff
@@ -1,11 +1,18 @@
 import { Component } from '@angular/core';
 
-
 @Component({
     selector:    'search',
     template: `
         <div class="searchContainer">
-            <input class="search" type="text" placeholder="search for courses" />
+            <input 
+                    class="search"
+                    #search
+                    type="text" 
+                    placeholder="search for courses"
+                    [value]="value"
```

Comment:
Why do you need `[value]` property binding, is `ngModel` not enough?

Reply:
ngModel is enought. fixed